Source/Urho3D/Graphics/Graphics.h

```diff
@@ -527,6 +527,9 @@ class URHO3D_API Graphics : public Object
     void SetVBO(unsigned object);
     /// Bind a UBO, avoiding redundant operation. Used only on OpenGL.
     void SetUBO(unsigned object);
+    
+    /// Return whether  the extention GL_OES_standard_derivatives is supported

```

Comment:
Doxygen comment should be terminated with a period.

Reply:
`whether  the` (double space)